readthedocs/projects/views/private.py

```diff
@@ -975,10 +1002,11 @@ class EnvironmentVariableList(EnvironmentVariableMixin, ListView):
 
 
 class EnvironmentVariableCreate(EnvironmentVariableMixin, CreateView):
-    pass
+    success_message = _("Environment Variable created")
```

Comment:
```suggestion
    success_message = _("Environment variable created")
```

I think Anthony would say that 😄